Create ui-c8y-1023-14-87-refactor-the-way-how-date-is-displayed-on-info-gauge.md (#4272)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-87-refactor-the-way-how-date-is-displayed-on-info-gauge.md

---
date: ""

> title: Info gauge now displays dates in consistent format with other dashboard elements

Reviewer: ```suggestion
title: Info gauge widget now displays dates in consistent format with other dashboard elements
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65447
version: 1023.14.87
---

> Previously, the info gauge widget displayed dates in a different format compared to other dashboard elements, creating inconsistency in how temporal information was presented across your dashboards. The date formatting has been standardized so that the info gauge now uses the same date format as all other dashboard components. This ensures a uniform and predictable experience when viewing date information throughout your dashboards, making it easier to read and interpret temporal data consistently regardless of which widget you are using.

Reviewer: ```suggestion
Previously, the "Info gauge" widget displayed dates in a different format than other dashboard elements, creating inconsistency in how temporal information was presented across your dashboards. The date formatting has been standardized so that the "Info gauge" widget now uses the same date format as other dashboard components. This ensures a uniform and predictable experience and makes it easier to read and interpret temporal data consistently.
```
